src/applications/vaos/containers/ReviewPage.jsx

```diff
@@ -74,6 +74,7 @@ export class ReviewPage extends React.Component {
               submitStatus === FETCH_STATUS.failed
             }
             isLoading={submitStatus === FETCH_STATUS.loading}
+            loadingText="Page change in progress"
```

Comment:
This one should be something like "Submission in progress"

Reply:
Done.